src/main/kotlin/org/jetbrains/intellij/utils.kt

```diff
@@ -283,7 +283,7 @@ private fun log(level: LogLevel, logCategory: String?, message: String, e: Throw
     }
 }
 
-fun Project.logCategory(): String = path + name.takeIf { ":$it" != path }.orEmpty()
+fun Project.logCategory(): String = path + if (path.subString(1) != name) " $name" else ""
```

Comment:
Note I was wondering if using `endsWith` would be better, but I'm unsure about kotlin compatility.

```suggestion
fun Project.logCategory(): String = path + if (path.endsWith(name)) " $name" else ""
```